src/main/kotlin/com/msg/gcms/domain/club/service/impl/CreateClubServiceImpl.kt

```diff
@@ -66,4 +69,8 @@ class CreateClubServiceImpl(
         if (clubDto.type != ClubType.EDITORIAL && applicantRepository.findAllByUser(currentUser).any { it.club.type == clubDto.type })
             throw AlreadyClubApplicantException()
     }
+
+    private fun sendDiscordMessageUtil(clubDto: ClubDto){
+        sendDiscordMessageUtil.execute(clubDto.name,clubDto.type,clubDto.bannerImg)
```

Comment:
```suggestion
        sendDiscordMessageUtil.execute(clubDto.name, clubDto.type, clubDto.bannerImg)
```
공백을 주어서 컨벤션을 맞추는게 좋을것 같아요?